--- foo.ly	2011-05-21 08:57:58.323358539 +0200
+++ foo-fixed.ly	2011-05-21 14:17:07.380475727 +0200
@@ -6,26 +6,28 @@
     title = "Sonata 3 in Bb Major"
     tagline = "Created using Rosegarden 11.02 and LilyPond"
 }
-#(set-global-staff-size 26)
+#(set-global-staff-size 22)
 #(set-default-paper-size "a4")
 global = { 
     \time 4/4
-    \skip 1*156  %% 1-156
+    % \skip 1*156  %% 1-156
 }
 globalTempo = {
-    \tempo 4 = 120  \skip 1*153 \skip 4 
-    \tempo 4 = 115  \skip 4 
-    \tempo 4 = 110  \skip 4 
-    \tempo 4 = 105  \skip 4 
-    \tempo 4 = 100  \skip 4 
-    \tempo 4 = 95  \skip 4 
-    \tempo 4 = 90  \skip 4 
-    \tempo 4 = 85  \skip 4 
-    \tempo 4 = 80  \skip 1 
+     \tempo 4 = 120  % \skip 1*153 \skip 4 
+%    \tempo 4 = 115  \skip 4 
+%    \tempo 4 = 110  \skip 4 
+%    \tempo 4 = 105  \skip 4 
+%    \tempo 4 = 100  \skip 4 
+%    \tempo 4 = 95  \skip 4 
+%    \tempo 4 = 90  \skip 4 
+%    \tempo 4 = 85  \skip 4 
+%    \tempo 4 = 80  \skip 1 
 }
 \score {
 << % common
 
+\set Score.currentBarNumber = #0
+
     \context Staff = "track 1, Rechts" << 
         \set Staff.instrumentName = \markup { \column { "Rechts " } }
         \set Staff.midiInstrument = ""
@@ -41,7 +43,10 @@
             \once \override Staff.TimeSignature #'style = #'() \time 4/4
             \clef "treble"
             \key bes \major
-            r2-\mp  r4 f'  |
+\override Rest #'transparent = ##t
+            r2  r4 f'-\mp  |
+\override Rest #'transparent = ##f
+\set Score.repeatCommands = #'(start-repeat)
             bes' 4. c'' 8 d'' 4 ees''  |
             f'' 4 ees'' 8 d'' c'' 4 bes'  |
             c'' 4 d'' 8 ees'' d'' 4 c''  |
@@ -60,7 +65,7 @@
             \skip 1*9 
 %% 10
             \clef "treble"
-            \key bes \major
+            %\key bes \major
             bes' 4 -\p c'' 8 d'' ees'' 4 f''  |
             g'' 4 f'' 8 ees'' d'' 4 c''  |
             d'' 4 ees'' 8 f'' ees'' 4 d''  |
@@ -107,7 +112,7 @@
 
             \skip 1*33 
             \clef "treble"
-            \key f \major
+            %\key f \major
             f' 8 -\p g' \< a' bes' c'' 2 
             % warning: overlong bar truncated here |
 %% 35
@@ -131,7 +136,9 @@
             % warning: overlong bar truncated here |
             d'' 8 e'' f'' g'' a'' 2  |
             e'' 8 f'' g'' a'' bes'' -\ff -\accent \! a'' \> f'' ees''  |
-            c'' 4 -\f -\accent \! \stemDown a' -\accent \stemNeutral bes' -\accent f' _\markup { \bold \italic "" } -\p r32 
+\set Score.repeatCommands = #'((volta "1."))
+            c'' 4 -\f -\accent \! \stemDown a' -\accent \stemNeutral bes' -\accent f' _\markup { \bold \italic "" } -\p
+%{
             % warning: overlong bar truncated here |
         } % Voice
 
@@ -232,14 +239,17 @@
 %% 95
             d'' 8 e'' f'' g'' a'' 2  |
             e'' 8 f'' g'' a'' bes'' -\ff -\accent \! a'' \> f'' ees''  |
-            c'' 4 -\f -\accent \! \stemDown a' -\accent \stemNeutral bes' 2 -\accent r32.  |
+%}
+\set Score.repeatCommands = #'((volta #f) (volta "2.") end-repeat)
+            c'' 4 -\f -\accent \! \stemDown a' -\accent \stemNeutral bes' 2 -\accent|
+\set Score.repeatCommands = #'((volta #f))
         } % Voice
 
         \context Voice = "voice 9" {
             \override Voice.TextScript #'padding = #2.0
             \override MultiMeasureRest #'expand-limit = 1
 
-            \skip 1*97 
+            \skip 1*50 
             \clef "treble"
             \key bes \major
             bes' 8 -\mp d'' f'' 4 ees'' 8 bes' g' 4  |
@@ -251,7 +261,7 @@
             c'' 8 f'' a'' 4 f'' 8 c'' bes' 4  |
             ees'' 8 g'' c''' 4 a' 8 c'' f'' 4  |
 %% 105
-            ees'' 8 d'' f'' f' bes' 4 f' r32 
+            ees'' 8 d'' f'' f' bes' 4 f'
             % warning: overlong bar truncated here |
         } % Voice
 
@@ -259,9 +269,9 @@
             \override Voice.TextScript #'padding = #2.0
             \override MultiMeasureRest #'expand-limit = 1
 
-            \skip 1*105 
+            \skip 1*58 
             \clef "treble"
-            \key bes \major
+            %\key bes \major
             bes' 4. -\mp c'' 8 d'' 4 ees''  |
             f'' 4 ees'' 8 d'' c'' 4 bes'  |
             c'' 4 d'' 8 ees'' d'' 4 c''  |
@@ -277,9 +287,9 @@
             \override Voice.TextScript #'padding = #2.0
             \override MultiMeasureRest #'expand-limit = 1
 
-            \skip 1*113 
+            \skip 1*66 
             \clef "treble"
-            \key bes \major
+            %\key bes \major
             bes' 4 -\p c'' 8 d'' ees'' 4 f''  |
 %% 115
             g'' 4 f'' 8 ees'' d'' 4 c''  |
@@ -306,7 +316,7 @@
             \override Voice.TextScript #'padding = #2.0
             \override MultiMeasureRest #'expand-limit = 1
 
-            \skip 1*129 
+            \skip 1*82 
 %% 130
             \clef "treble"
             \key bes \major
@@ -325,9 +335,9 @@
             \override Voice.TextScript #'padding = #2.0
             \override MultiMeasureRest #'expand-limit = 1
 
-            \skip 1*137 
+            \skip 1*90 
             \clef "treble"
-            \key bes \major
+            %\key bes \major
             bes' 8 -\p c'' \< d'' ees'' f'' 2 
             % warning: overlong bar truncated here |
             c'' 8 d'' ees'' f'' g'' 2  |
@@ -376,7 +386,10 @@
             \once \override Staff.TimeSignature #'style = #'() \time 4/4
             \clef "bass"
             \key bes \major
-            R1-\p   |
+\override Rest #'transparent = ##t
+	    r2 r4
+\override Rest #'transparent = ##f
+            r4-\p   |
             bes, 8 f d' 4 bes 8 f d 4  |
             c 8 f a 4 f 8 c bes, 4  |
             ees 8 g c' 4 a 8 f c 4  |
@@ -385,7 +398,7 @@
             bes, 8 f d' 4 bes 8 f d 4  |
             c 8 f a 4 f 8 c bes, 4  |
             ees 8 g c' 4 a, 8 c f 4  |
-            ees 8 d f f, bes, 2 r32.  |
+            ees 8 d f f, bes, 2|
         } % Voice
 
         \context Voice = "voice 15" {
@@ -395,7 +408,7 @@
             \skip 1*9 
 %% 10
             \clef "bass"
-            \key g \minor
+            %\key bes \major
             bes, 8 -\p f bes d' < f bes d' > 2  |
             g, 8 g bes d' < bes d' g > 2  |
             d 8 f a d' < d' f a > 2  |
@@ -415,7 +428,7 @@
             a, 8 a c' e' < e' c' a > 2  |
             g, 8 e g c' < f a c > 2  |
 %% 25
-            c 8 g bes e f 2 r32.  |
+            c 8 g bes e f 2|
         } % Voice
 
         \context Voice = "voice 16" {
@@ -433,7 +446,7 @@
             f 8 a c' 4 b 8 gis e 4  |
             c 8 e a 4 f 8 bes d 4  |
             c 8 f a 4 f 8 a c' 4  |
-            c' 8 g bes e f 2 r32.  |
+            c' 8 g bes e f 2|
         } % Voice
 
         \context Voice = "voice 17" {
@@ -442,7 +455,7 @@
 
             \skip 1*33 
             \clef "bass"
-            \key f \major
+            %\key f \major
             < c' a f > 2 -\p e 8 g c' e'  |
 %% 35
             < g c' e' > 2 d 8 g bes d'  |
@@ -462,7 +475,8 @@
             < e c g > 2 b, 8 d g b  |
             < d b g > 2 a, 8 d f a  |
             < a f d > 2 < a ees c' >  |
-            c 8 ees f f, bes, 2 r32.  |
+            c 8 ees f f, bes, 2|
+%{
         } % Voice
 
         \context Voice = "voice 18" {
@@ -558,14 +572,15 @@
 %% 95
             < d b g > 2 a, 8 d f a  |
             < a f d > 2 < a ees c' >  |
-            c 8 ees f f, bes, 2 r32.  |
+%}
+            c 8 ees f f, bes, 2|
         } % Voice
 
         \context Voice = "voice 22" {
             \override Voice.TextScript #'padding = #2.0
             \override MultiMeasureRest #'expand-limit = 1
 
-            \skip 1*97 
+            \skip 1*50 
             \clef "bass"
             \key bes \major
             bes 4 -\mp g 8 a bes 4 g  |
@@ -584,9 +599,9 @@
             \override Voice.TextScript #'padding = #2.0
             \override MultiMeasureRest #'expand-limit = 1
 
-            \skip 1*105 
+            \skip 1*58 
             \clef "bass"
-            \key bes \major
+            %\key bes \major
             bes, 8 -\p f d' 4 bes 8 f d 4  |
             c 8 f a 4 f 8 c bes, 4  |
             ees 8 g c' 4 a 8 f c 4  |
@@ -595,16 +610,16 @@
             bes, 8 f d' 4 bes 8 f d 4  |
             c 8 f a 4 f 8 c bes, 4  |
             ees 8 g c' 4 a, 8 c f 4  |
-            ees 8 d f f, bes, 2 r32.  |
+            ees 8 d f f, bes, 2|
         } % Voice
 
         \context Voice = "voice 24" {
             \override Voice.TextScript #'padding = #2.0
             \override MultiMeasureRest #'expand-limit = 1
 
-            \skip 1*113 
+            \skip 1*66 
             \clef "bass"
-            \key g \minor
+            %\key bes \major
             bes, 8 -\p f bes d' < f bes d' > 2  |
 %% 115
             g, 8 g bes d' < bes d' g > 2  |
@@ -624,14 +639,14 @@
             c 8 g c' e' < g e' c' > 2  |
             a, 8 a c' e' < e' c' a > 2  |
             c 8 f a c' < f bes d > 2  |
-            a 8 c f f, bes, 2 r32.  |
+            a 8 c f f, bes, 2|
         } % Voice
 
         \context Voice = "voice 25" {
             \override Voice.TextScript #'padding = #2.0
             \override MultiMeasureRest #'expand-limit = 1
 
-            \skip 1*129 
+            \skip 1*82 
 %% 130
             \clef "bass"
             \key bes \major
@@ -643,16 +658,16 @@
 %% 135
             f, 8 a, d 4 bes, 8 ees g, 4  |
             f, 8 bes, d 4 bes, 8 d f 4  |
-            f 8 c ees a, bes, 2 r32.  |
+            f 8 c ees a, bes, 2|
         } % Voice
 
         \context Voice = "voice 26" {
             \override Voice.TextScript #'padding = #2.0
             \override MultiMeasureRest #'expand-limit = 1
 
-            \skip 1*137 
+            \skip 1*90 
             \clef "bass"
-            \key bes \major
+            %\key bes \major
             < f d bes, > 2 -\p a, 8 c f a  |
             < c f a > 2 g, 8 c ees g  |
 %% 140
