--- sonatina8.ly.orig	2016-12-08 20:57:07.963371057 -0500
+++ sonatina8.ly	2016-12-08 21:05:56.535386600 -0500
@@ -6,17 +6,15 @@
     title = "Sonatina 8 in C"
     tagline = "Created using Rosegarden 14.02 and LilyPond"
 }
-#(set-global-staff-size 20)
+#(set-global-staff-size 18)
 #(set-default-paper-size "a4")
 global = { 
     \time 4/4
-    \skip 1*290  %% 1-290
+    \skip 1*210  %% 1-290
 }
 globalTempo = {
     \tempo 4 = 150  \skip 1*40 \skip 1*8 
     \tempo 4 = 210  \skip 1*7 \skip 1 
-    \tempo 4 = 150  \skip 1*64 \skip 1*8 
-    \tempo 4 = 210  \skip 1*7 \skip 1 
     \tempo 4 = 150  \skip 1*112 \skip 1*8 
     \tempo 4 = 180  \skip 1*7 \skip 1 
     \tempo 4 = 150  \skip 1*21 \skip 1*5 
@@ -96,6 +94,7 @@
                 d' 4. g' bes' 4  |
                 < d' fis' a' > 2 -\f \! g' 4 \> fis'  |
                 < g' d' bes > 1 -\mf \!  |
+\bar "||"
             } % Voice
 
             \context Voice = "voice 3" {
@@ -104,7 +103,7 @@
 
                 \skip 1*32 
                 \clef "treble"
-                \key g \minor
+                % \key g \minor
                 g' 4. -\mf a' 8 bes' 4 c''  |
                 d'' 4 ees'' d'' 2  |
 %% 35
@@ -153,7 +152,7 @@
                 \skip 1*64 
 %% 65
                 \clef "treble"
-                \key g \minor
+                % \key g \minor
                 g' 8 -\mf -\mf bes' d'' bes' g' bes' d'' bes'  |
                 g' 8 c'' ees'' c'' g' c'' ees'' c''  |
                 d' 8 f' d'' f' d' f' d'' f'  |
@@ -176,9 +175,10 @@
 %% 80
                 \key c \major
                 < g' c'' e' > 1 -\f  |
-                 |
+\bar ":|."
             } % Voice
 
+%{
             \context Voice = "voice 5" {
                 \override Voice.TextScript #'padding = #2.0
                 \override MultiMeasureRest #'expand-limit = 1
@@ -318,12 +318,13 @@
                 < g' c'' e' > 1 -\f  |
                  |
             } % Voice
+%}
 
             \context Voice = "voice 9" {
                 \override Voice.TextScript #'padding = #2.0
                 \override MultiMeasureRest #'expand-limit = 1
 
-                \skip 1*160 
+                \skip 1*80 
                 \clef "treble"
                 c' 4. -\mf e' g' 4  |
                 f' 4. e' d' 4  |
@@ -390,14 +391,14 @@
                 % warning: overlong bar truncated here |
                 b' 8 f' g' r b r r4-\p  \!  |
                 < e' c'' > 4 -\f < g' e' > < e' c' > c'  |
-                 |
+\bar "||"
             } % Voice
 
             \context Voice = "voice 10" {
                 \override Voice.TextScript #'padding = #2.0
                 \override MultiMeasureRest #'expand-limit = 1
 
-                \skip 1*208 
+                \skip 1*128 
                 \clef "treble"
                 \key c \major
                 c' 4. -\mf ^\markup { \bold \large "Allegro" } e' g' 4  |
@@ -425,7 +426,7 @@
                 \override Voice.TextScript #'padding = #2.0
                 \override MultiMeasureRest #'expand-limit = 1
 
-                \skip 1*224 
+                \skip 1*144 
 %% 225
                 \clef "treble"
                 c' 4. -\mp e' g' 4  |
@@ -450,13 +451,14 @@
                 < g' d' d'' > 2 -\mf \! c'' 4 b'  |
 %% 240
                 < c'' g' e' > 1  |
+\bar "||"
             } % Voice
 
             \context Voice = "voice 12" {
                 \override Voice.TextScript #'padding = #2.0
                 \override MultiMeasureRest #'expand-limit = 1
 
-                \skip 1*240 
+                \skip 1*160 
                 \clef "treble"
                 c' 4. -\mf d' 8 e' 4 f'  |
                 g' 4 a' g' 2  |
@@ -503,7 +505,7 @@
                 \override Voice.TextScript #'padding = #2.0
                 \override MultiMeasureRest #'expand-limit = 1
 
-                \skip 1*272 
+                \skip 1*192 
                 \clef "treble"
                 c' 8 -\mf e' g' e' c' e' g' e'  |
                 c' 8 f' a' f' c' f' a' f'  |
@@ -601,6 +603,7 @@
                 g,, 8 g, bes, g,, g, bes, d g  |
                 < a, d, > 2 -\f \! d 4 \> d,  |
                 g, 1 -\mf \!  |
+\bar "||"
             } % Voice
 
             \context Voice = "voice 16" {
@@ -609,7 +612,7 @@
 
                 \skip 1*32 
                 \clef "bass"
-                \key g \minor
+                % \key g \minor
                 < g, g d > 1 -\mf  |
                 < g, bes d' > 4 < c g ees > < d f d' > 2  |
 %% 35
@@ -658,7 +661,7 @@
                 \skip 1*64 
 %% 65
                 \clef "bass"
-                \key g \minor
+                % \key g \minor
                 g, 2. -\mf bes, 4  |
                 c 1  |
                 d 2. d 4  |
@@ -681,9 +684,10 @@
 %% 80
                 \key c \major
                 < c g > 1 -\f  |
-                 |
+\bar ":|."
             } % Voice
 
+%{
             \context Voice = "voice 18" {
                 \override Voice.TextScript #'padding = #2.0
                 \override MultiMeasureRest #'expand-limit = 1
@@ -822,12 +826,13 @@
                 < c g > 1 -\f  |
                  |
             } % Voice
+%}
 
             \context Voice = "voice 22" {
                 \override Voice.TextScript #'padding = #2.0
                 \override MultiMeasureRest #'expand-limit = 1
 
-                \skip 1*160 
+                \skip 1*80 
                 \clef "bass"
                 c, 8 -\mf g, c c, g, c e g  |
                 c, 8 f, c c, f, c f a  |
@@ -894,14 +899,14 @@
                 % warning: overlong bar truncated here |
                 r4 r8 g r b, d -\p \! g,  |
                 < c g > 4 -\f < g c > < g c > < e c g >  |
-                 |
+\bar "||"
             } % Voice
 
             \context Voice = "voice 23" {
                 \override Voice.TextScript #'padding = #2.0
                 \override MultiMeasureRest #'expand-limit = 1
 
-                \skip 1*208 
+                \skip 1*128 
                 \clef "bass"
                 \key c \major
                 < c, c e > 1 -\mf  |
@@ -929,7 +934,7 @@
                 \override Voice.TextScript #'padding = #2.0
                 \override MultiMeasureRest #'expand-limit = 1
 
-                \skip 1*224 
+                \skip 1*144 
 %% 225
                 \clef "bass"
                 c, 8 -\mp g, c c, g, c e g  |
@@ -954,13 +959,14 @@
                 < d g, > 2 -\mf \! g 4 g,  |
 %% 240
                 c 1  |
+\bar "||"
             } % Voice
 
             \context Voice = "voice 25" {
                 \override Voice.TextScript #'padding = #2.0
                 \override MultiMeasureRest #'expand-limit = 1
 
-                \skip 1*240 
+                \skip 1*160 
                 \clef "bass"
                 < c, c g, > 1 -\mf  |
                 < c, e g > 4 < f, c a, > < g, b, g > 2  |
@@ -1007,7 +1013,7 @@
                 \override Voice.TextScript #'padding = #2.0
                 \override MultiMeasureRest #'expand-limit = 1
 
-                \skip 1*272 
+                \skip 1*192 
                 \clef "bass"
                 c 2. -\mf e 4  |
                 f 1  |
