--- sonatina6.ly.orig	2014-04-07 20:10:03.686591614 +0200
+++ sonatina6.ly	2014-04-07 20:24:11.351027488 +0200
@@ -8,12 +8,15 @@
 }
 #(set-global-staff-size 20)
 #(set-default-paper-size "a4")
+\paper {
+    system-system-spacing #'padding = #3
+}
 global = { 
     \time 4/4
     
 }
 globalTempo = {
-    \tempo 4 = 90  \skip 1*9 \skip 1 \skip 1*15 
+    \tempo 4 = 90  \skip 1*9 \skip 1 \skip 1*15 %{
     \override Score.MetronomeMark #'transparent = ##t
     \tempo 4 = 105  \skip 1 \skip 1*8 \skip 1*8 
     \tempo 4 = 90  \skip 1*15 \skip 1 \skip 1*15 
@@ -23,6 +26,7 @@
     \tempo 4 = 90  \skip 1 \skip 1*25 \skip 1 \skip 1*15 
     \tempo 4 = 105  \skip 1 \skip 1*8 \skip 1*8 
     \tempo 4 = 90  \skip 1*7 \skip 1 \skip 1 \skip 1 
+    %}
 }
 \score {
 << % common
@@ -347,7 +351,7 @@
                 < d b, g > 1 \< 
                 % warning: overlong bar truncated here |
                 < a f d > 1  |
-                < cis ais, fis > 1 -\f \! e 2 -\accent  |
+                << { \voiceOne fis 2 -\f \! e 2 -\accent } \new Voice { \voiceTwo < cis ais, > 1 } >> \oneVoice |
                 \key h \minor
                 < fis, d h, > 1 ^\markup { \bold "Moderato" } \> 
                 % warning: overlong bar truncated here |
